common/xmlparser.h

```diff
@@ -348,6 +368,7 @@ class XMLParser {
 	char _char;
 	SeekableReadStream *_stream;
 	String _fileName;
+	bool _allowText; /** Allow text nodes in the doc (default false) */
```

Comment:
I would move this below _char to reduce the size of the class

Reply:
sure, done.